examples/wasm/index.html

```diff
@@ -1,9 +1,27 @@
 <html>
   <head>
     <meta charset="UTF-8" />
+    <style>
+      body {
+        background: linear-gradient(
+          135deg,
+          white 0%,
+          white 49%,
+          black 49%,
+          black 51%,
+          white 51%,
+          white 100%
+        );
+        background-repeat: repeat;
+        background-size: 20px 20px;
+      }
+      canvas {
+        background-color: white;
+      }
+    </style>
   </head>
   <script type="module">
-    import init from './target/headless_wasm.js'
+    import init from './target/winit_wasm.js'
```

Comment:
The [build & run](https://github.com/bevyengine/bevy/tree/master/examples#build--run) instructions should be updated to build `winit_wasm` instead of `headless_wasm`; otherwise, it will lead to module loading errors in browsers by default. We should also mention that `--features bevy_winit` is needed.